__docs__/wonder-blocks-icon/accessibility.stories.mdx

```diff
@@ -0,0 +1,99 @@
+import {Meta, Story, Canvas} from "@storybook/addon-docs";
+
+import Color from "@khanacademy/wonder-blocks-color";
```

Comment:
**suggestion:** It looks like `import {View} from wb-core` is missing here. I was inspecting the HTML and noticed that the `View` styles defined in this file are not being applied correctly, so I'm not sure if this is related to the missing import.

<img width="1137" alt="Screen Shot 2023-03-14 at 10 50 50 AM" src="https://user-images.githubusercontent.com/843075/225039809-a4061da5-74b5-49f8-8105-84f9cb94fa85.png">

Reply:
Oh!! Good catch!